<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugGSSettingsTab</class>
 <widget class="QWidget" name="DebugGSSettingsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Draw Dumping</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="dumpGSData">
          <property name="text">
           <string>Dump GS Draws</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="saveRT">
          <property name="text">
           <string>Save RT</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="saveFrame">
          <property name="text">
           <string>Save Frame</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="saveTexture">
          <property name="text">
           <string>Save Texture</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="saveDepth">
          <property name="text">
           <string>Save Depth</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="saveAlpha">
          <property name="text">
           <string>Save Alpha</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="saveInfo">
          <property name="text">
           <string>Save Info</string>
          </property>
         </widget>
        </item>
		<item row="4" column="0">
		  <widget class="QCheckBox" name="saveDrawStats">
		   <property name="text">
		    <string>Save Draw Stats</string>
		   </property>
		  </widget>
		</item>
		<item row="4" column="1">
		  <widget class="QCheckBox" name="saveFrameStats">
		    <property name="text">
		      <string>Save Frame Stats</string>
		    </property>
		  </widget>
		</item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="saveTransferImages">
          <property name="text">
           <string>Save Transfer Image Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Save Draw Start:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="saveDrawStart">
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Save Draw Count:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="saveDrawCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Save Frame Start:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="saveFrameStart">
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Save Frame Count:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="saveFrameCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Hardware Dump Directory:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Software Dump Directory:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
        <item>
         <widget class="QLineEdit" name="hwDumpDirectory"/>
        </item>
        <item>
         <widget class="QPushButton" name="hwDumpBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="hwDumpOpen">
          <property name="text">
           <string>Open...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
        <item>
         <widget class="QLineEdit" name="swDumpDirectory"/>
        </item>
        <item>
         <widget class="QPushButton" name="swDumpBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="swDumpOpen">
          <property name="text">
           <string>Open...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
